code/notificationbuilder/src/test/java/com/adobe/marketing/mobile/notificationbuilder/internal/templates/MockDataUtils.kt

```diff
@@ -20,7 +20,7 @@ const val MOCKED_BODY = "Mocked Body"
 const val MOCKED_PAYLOAD_VERSION = "1"
 const val MOCKED_CAROUSEL_LAYOUT = "default"
 const val MOCKED_BODY_TEXT_COLOR = "#FFFFFF"
-const val MOCKED_SMALL_ICON = "chat_bubble"
+const val MOCKED_SMALL_ICON = "skipleft"
```

Comment:
Is this change intentional? Does this refer to an asset?

Reply:
Yes this is an intentional change.

Reply:
we need this since this asset is available in the resource directory